perses/app/fah_generator.py

```diff
@@ -0,0 +1,203 @@
+#1/usr/bin/env python
+
+__author__ = 'dominic rufa'
+
+"""
+Folding@Home perses executor
+
+# prereqs
+conda config --add channels omnia --add channels conda-forge
+conda create -n perses python3.7 perses tqdm dicttoxml
+pip uninstall --yes openmmtools
+pip install git+https://github.com/choderalab/openmmtools.git
+
+argv[1]: setup.yaml (argument for perses.app.setup_relative_calculation.getSetupOptions)
+argv[2]: neq_setup.yaml (contains keywords for openmmtools.integrators.PeriodicNonequilibriumIntegrator arguments)
+argv[3]: run_number (project run number; defined by f"setup_options['trajectory_directory']_phase/RUN_{run_number}")
+"""
+import yaml
+import numpy as np
+import pickle
+import os
+import sys
+import simtk.unit as unit
+from simtk import openmm
+from perses.app.setup_relative_calculation import getSetupOptions, run_setup
+from perses.annihilation.relative import HybridTopologyFactory
+from perses.app.relative_setup import RelativeFEPSetup
+from perses.annihilation.lambda_protocol import LambdaProtocol
+from perses.dispersed.feptasks import minimize
+from openmmtools import SamplerState, ThermodynamicState, CompoundThermodynamicState
+
+from openmmtools import mcmc, utils
+from perses.utils.smallmolecules import render_atom_mapping
+from perses.tests.utils import validate_endstate_energies
+from perses.dispersed.smc import SequentialMonteCarlo
+from simtk.openmm import XmlSerializer
+from perses.utils import data
+from openmmtools.integrators import PeriodicNonequilibriumIntegrator
+from copy import deepcopy
+import tqdm
+from openmmtools.integrators import LangevinIntegrator
+
+import datetime
+class TimeFilter(logging.Filter):
+    def filter(self, record):
+        try:
+          last = self.last
+        except AttributeError:
+          last = record.relativeCreated
+        delta = datetime.datetime.fromtimestamp(record.relativeCreated/1000.0) - datetime.datetime.fromtimestamp(last/1000.0)
+        record.relative = '{0:.2f}'.format(delta.seconds + delta.microseconds/1000000.0)
+        self.last = record.relativeCreated
+        return True
+
+fmt = logging.Formatter(fmt="%(asctime)s:(%(relative)ss):%(name)s:%(message)s")
+#logging.basicConfig(level = logging.NOTSET)
+logging.basicConfig(
+    format='%(asctime)s %(levelname)-8s %(message)s',
+    level=logging.INFO,
+    datefmt='%Y-%m-%d %H:%M:%S')
+_logger = logging.getLogger()
+_logger.setLevel(logging.INFO)
+[hndl.addFilter(TimeFilter()) for hndl in _logger.handlers]
+[hndl.setFormatter(fmt) for hndl in _logger.handlers]
+
+setup_yaml = sys.argv[1]
```

Comment:
As a general practice, let's be sure to at least use [`argparse`](https://docs.python.org/3/library/argparse.html), and never use `sys.argv`. 

This should also always occur inside
```python
if __name__ == "__main__":
    ...
```
so that we can import methods from this module if we need to reuse them.